app-editors/emacs/emacs-29.2-r2.ebuild

```diff
@@ -586,6 +586,14 @@ src_install() {
 			\"${EPREFIX}/Applications/Gentoo\". You may want to copy or
 			symlink it into /Applications by yourself."
 	fi
+	if ! use mailutils; then
+		DOC_CONTENTS+="The mailutils USE flag is disabled.\\n If emacs'
+		own email features are going to be used as an email client
+		(rmail, gnus), you are strongly encouraged to enable it. If not,
```

Comment:
Following @xgqt 's comments, I did some researching on which packages actually use the `movemail` program.
Rmail is explicitly mentioned in the documentation; however, I do believe gnus also uses movemail. 
Here's what I believe to be Emacs default movemail implementation and the path that led me to it.

- https://github.com/emacs-mirror/emacs/blob/master/lisp/gnus/mail-source.el#L31
- https://github.com/emacs-mirror/emacs/blob/master/lisp/net/pop3.el#L39
- https://github.com/emacs-mirror/emacs/blob/master/lisp/mail/mail-utils.el

Reply:
GNUS can fetch mail from the mailserver itself using IMAP, so normally it doesn't need movemail.